mwdb/web/src/components/Settings/Views/AccessControl.js

```diff
@@ -0,0 +1,288 @@
+                </ShowIf>
+            </th>
+        </tr>
+    );
+}
+
+function CapabilitiesList({ capabilities, onDelete }) {
+    return capabilities.map((cap) => (
+        <tr>
+            <td className="col-auto">
+                <Link
+                    onClick={(ev) => {
+                        ev.preventDefault();
+                        onDelete(cap);
+                    }}
+                >
+                    <FontAwesomeIcon icon={faTimes} />
+                </Link>
+            </td>
+            <th className="col-1">
+                <span className="badge badge-success">{cap}</span>
+            </th>
+            <td className="col">
+                {capabilitiesList[cap] ||
+                    "Undocumented. Possibly internal and required by plugin."}
+            </td>
+        </tr>
+    ));
+}
+
+function CapabilityChangeCard({ groups, onSubmit }) {
+    const [groupName, setGroupName] = useState("");
+    const [chosenGroup, setChosenGroup] = useState({});
+    const [chosenCapabilities, setChosenCapabilities] = useState();
+
+    const capabilities = Object.keys(capabilitiesList);
+    const selectedCaps = chosenCapabilities || [];
+    const originalCaps = chosenGroup.capabilities || [];
+    const changedCaps = capabilities.filter(
+        (cap) => selectedCaps.includes(cap) !== originalCaps.includes(cap)
+    );
+    const selectHandler = useCallback(
+        (e, clickedIndex, isSelected) => {
+            if (isSelected)
+                setChosenCapabilities(
+                    selectedCaps.concat(capabilities[clickedIndex])
+                );
+            else
+                setChosenCapabilities(
+                    selectedCaps.filter(
+                        (cap) => cap !== capabilities[clickedIndex]
+                    )
+                );
+        },
+        [selectedCaps, capabilities]
+    );
+
+    function dismissChanges() {
+        setChosenCapabilities(chosenGroup.capabilities);
+    }
+
+    useEffect(() => {
+        const matchedGroup =
+            groups.find((group) => group.name === groupName) || {};
+        setChosenGroup(matchedGroup);
+        setChosenCapabilities(matchedGroup.capabilities);
+    }, [groups, groupName]);
+
+    return (
+        <div className="card">
+            <div className="card-body">
+                <Autocomplete
+                    value={groupName}
+                    getItemValue={(group) => group.name}
+                    items={groups.filter(
+                        (group) =>
+                            group.name
+                                .toLowerCase()
+                                .indexOf(groupName.toLowerCase()) !== -1
+                    )}
+                    onChange={(value) => setGroupName(value)}
+                    className="form-control"
+                    placeholder="Group name"
+                    renderItem={({ item }) => <GroupBadge group={item} />}
```

Comment:
GroupBadge component here is quite confusing, the problem is that when we click on the badge we are redirected to the group or profile view, which is not always intended. The badge is a part of select in autocomplete so it could be easy click by accident. Maybe add argument to GroupBadge that prevent from redirecting?

Reply:
Agree, added `clickable` argument to GroupBadge to prevent that: https://github.com/CERT-Polska/mwdb-core/pull/356/commits/893775a2f29921a98cb45528e858784e41fe029b